### DISCUSSION TITLE
Liquidsoap related: Force meta data delay?

### DISCUSSION THREAD

**Discussion by RM-FM** [AzuraCast/AzuraCast#6715] [General]
Hi,

Is there a way to delay the meta data for e.g. 2 seconds in Liquidsoap? I know it is possible to manipulate the meta data content wise. Does anyone know whether it is also possible to delay the meta data on track change?

The goal is to sync the ad injection, which is based on meta data detection, with the audio. Currently the injection cuts the song approx. 3 sec. before the actual end.

**Comment by BusterNeece** [comment]
This is probably a question best answered over on Liquidsoap's GitHub (https://github.com/savonet/liquidsoap) or on their community Discord.
